### ISSUE TITLE
OutRyder Tournament Format Support

### ISSUE THREAD

**Issue by kingargyle** [Killerardvark/CryodexSource#38]
This is a feature request to have an OutRyder tournament format supported.   I'll tag Ryan Flemming who has developed a spreadsheet for this with the business rules, but the spreadsheet is starting to reach it's limits, so thought it would be good to have the requirements in Cryodex.

**Comment by Killerardvark** [comment]
I realize we missed this for 2017, but we can start looking into this for 2018.

**Comment by kingargyle** [comment]
Thanks.   I think the next OutRyder Cup will be January 2018 at the earliest, it all depends on when Regionals fall, and System Opens.

The basics are as follows:

1. Teams - made up of 6 players.

2. 3 Rounds of play.
    a.  Round 1 - Epic Round, 300 pts, each player plays.
    b.  Round 2 and 3 - standard 100 pt rounds.

3. Scoring - wins and losses are tracked as normal, with 1 point for a win, and 0 for a loss for each person.  The team earns points for each person on the team.   So if a 4 team members, win, the team has 4 points that round.
     a. Ties or Modified Wins, are skins... the points for that game would carry over to the next round.
     b.  If there is a modified win, at the end of the last round (round 3), then no points are earned.

4.  Pairings are done a bit different then the standard parings each round.   Team Captains and Co-Captains match up the lists.   They pick and choose, so there will be some way to manually enter the pairings once this is finished.  This happens before each round of play begins.

5. Mov is commulative for a team.  Take all team members mov, and add it to the team total.    The epic mov is divided by a 1/3  so as not to throw it totally out of balance compared to the standard rounds.

Note that it would be nice to have a way to select the format of each round.   This way people could choose to say do a sub tournament of Escalation instead of an epic tournament, or they could just do a standard round in place of the epic round.

I think that covers the basics for scoring, Ryan Flemming would have more details.

**Comment by kingargyle** [comment]
:(
